RealmSwift/Optional.swift

```diff
@@ -74,13 +74,12 @@ extension RealmOptional: Equatable where Value: Equatable {
 }
 
 @available(*, deprecated, message: "RealmOptional has been deprecated, use RealmProperty<T?> instead.")
-extension RealmOptional: Codable where Value: Codable {
+extension RealmOptional: Codable where Value: Codable, Value: _RealmSchemaDiscoverable {
```

Comment:
Why has this sneaked in to the PR?

Reply:
This changes are already in master, I don't know why they are showing on this pull request; I rebase my barch with a master a week ago, any clues why @jsflax ?